engine/common/system.c

```diff
@@ -581,6 +590,22 @@ qboolean Sys_NewInstance( const char *gamedir )
 	Host_Shutdown( );
 	envSetNextLoad( exe, newargs );
 	exit( 0 );
+#elif XASH_PSVITA
+	const char *exe = "app0:/eboot.bin";
+	char newgamedir[256];
+	char *newargv[4];
+	int newargc = 0;
+	// make a copy of the gamedir name just in case
+	Q_strncpy( newgamedir, gamedir, sizeof( newgamedir ) );
+	// TODO: carry over the old args
+	newargv[newargc++] = host.argv[0]; // this is not usually the executable path
+	newargv[newargc++] = (char *)"-game";
+	newargv[newargc++] = newgamedir;
+	newargv[newargc] = NULL;
```

Comment:
Is it ok that we're losing old arguments? There is no way for them to be entered, right?

Reply:
Right now none are passed to the executable at all. The only way to pass arguments on the Vita is either through a separate launcher application or through custom LiveArea buttons, so if those are ever added this will need to be fixed.

Reply:
Fixed in 5f06b73.